### ISSUE TITLE
Per-database limits for max size and table limits

### ISSUE THREAD

**Issue by snaildos** [pterodactyl/panel#3345]
**Is your feature request related to a problem? Please describe.**
Being able to set a database limit per server. (MAX tables, or max size. Not sure.)

**Describe the solution you'd like**
It be cool to auto monitor databases that are created and allow to set limits.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by Software-Noob** [comment]
There is already database limit per server. It's by default set to 0, you can adjust it in the build configuration.

**Comment by FoksVHox** [comment]
This is already possible?

**Comment by DaneEveritt** [comment]
Think he is looking for limits on each individual database, not just the number of them.

**Comment by FoksVHox** [comment]
Ohhh

**Comment by snaildos** [comment]
Yes, the limits on each database. Otherwise, it's pretty much "unlimited database" 😓

**Comment by ririko5834** [comment]
Good idea

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
